tools/scxtop/src/keymap.rs

```diff
@@ -97,4 +113,264 @@ impl KeyMap {
             .collect::<Vec<_>>()
             .join("/")
     }
+
+    pub fn to_hashmap(&self) -> HashMap<String, String> {
+        let mut map = HashMap::new();
+        for (key, action) in &self.bindings {
+            map.insert(format!("{}", key), format!("{}", action));
+        }
+        map
+    }
```

Comment:
nit: I think this function should probably be renamed. `to_hashmap` implies it's the same as `Into<HashMap<_, _>>` but this is really doing some formatting. Maybe `format_map` or something?